data/human/war reactions.txt

```diff
@@ -0,0 +1,248 @@
+			`	"Only the usual assortment of hoaxes," says Rain. The sisters both laugh at this.`
+			`	"Seriously, though, both the Free Worlds Senate and Republic Parliament are up in arms over fear of aliens," says Fiona. Rain adds, "Only metaphorically up in arms, this time.`
+			`	"If we find new aliens, we'll let the necessary authorities know as soon as possible. You should do the same. Be discrete, avoid conflicts, and seek help if you need it."`
+				goto questions
+			label navy
+			action
+				set "War Reactions: Navy and Free Worlds Captains: label navy"
+			`	"Joint police operation," says Captain Rain. "It's more common than you may think."`
+			`	Captain Fiona nods. "We're shutting down a galactic Spider Funk smuggling operation. It's an illegal hallucinogenic. Not addictive, but still dangerous. The organization running it is huge, well funded, and well armed."`
+			`	"Not a match for a Dreadnought and an Auxiliary, though," brags Rain.`
+			`	"We have to fly all over Republic and Free Worlds space, so we have a joint patrol with one ship from each state."`
+			`	"Of course, we have another mission," says Fiona. "Everywhere we go, we look out for new alien activity."`
+				goto questions
+			label sister
+			action
+				set "War Reactions: Navy and Free Worlds Captains: label sister"
+			`	"Yup," Fiona says, "I blew up her Cruiser! When her ship's escape pods were veering towards a star, we rescued them."`
+			`	Rain continues, "Can you imagine my surprise when I woke up on my sister's new Dreadnought?"`
+			`	"Hey, you got a new Auxiliary commission afterwards," replies Fiona, poking Rain as she says it, "and your whole crew survived. Now we escort each other's ships during investigations!"`
+				goto questions
+			label mercenary
+			action
+				set "War Reactions: Navy and Free Worlds Captains: label mercenary"
+			`	"I meant no offense," says Fiona, "you're quite the hero." She thinks for a second and glances at her sister. "Rain, can you explain?"`
+			`	Rain says, "We're uniformed career officers who must follow orders. Our states own the ships, uniforms, and everything else we use. Our salaries may be meager compared to your earnings, but serving one's state is a rich reward.`
+			`	"You, on the other hand, own your ship and serve whoever you want. To keep up with costs, you have to charge a lot of money for your services. Mercenaries only survive as long as you if they're skilled at both finances and combat."`
+				goto questions
+
+
+mission "War Reactions: Syndicate Orphan"
+	landing
+	minor
+	source
+		government "Syndicate"
+		not attributes "station"
+	to offer
+		has "event: post-war reactions begin"
+		random < 30
+	on offer
+		set "pug media coverage"
+		conversation
+			`	When you exit your ship, you hear sounds of fists falling on skin and a kid yelling for help. Around the corner are three boys hitting a fourth, who just fell on the ground. It's not clear if they're going to stop attacking him.`
```

Comment:
This situation is nonsensical near to any kind of managed landing zone.

Reply:
I'm gonna disagree with that. I've seen this and worse among older children/teens, both as an adult recently and while a child myself, even in well maintained/regulated areas and even in the direct presence of adults, many of whom did exactly `(Ignore them and keep walking.)`

Reply:
Judging by the missions I've read, the Syndicate spaceports are rife with smuggling and bribery. It's perfectly reasonable for kids to be able to sneak into places they shouldn't be. As long as the kids aren't messing with the spaceships, many adults won't care. This may seem out-of-place in a shiny first-world country, but in other places, the rules aren't quite so simple.

Reply:
The mission now begins with, "When you exit the secure landing area..."

Reply:
That seems fine. To a degree I imagine a spaceport of any meaningful size is more like an airport. You have to actually disembark *into* the spaceport. Not merely step off your ship and "walk around a corner".
Of course once you're *in* the spaceport yeah I don't see a lot of issue with stuff like that happening there.

Reply:
This can happen at any Syndicate planet, so I'm trying to be vague about the relative positions of things.